extension/scripts/sync.js

```diff
@@ -247,6 +350,54 @@ const DEBUG = false;
                   data.payload.keyItems.forEach(keyItem => {
                     if (!App.game.keyItems.hasKeyItem(keyItem)) {
                       App.game.keyItems.gainKeyItem(keyItem)
+                      if (keyItem == KeyItems.KeyItem.Dungeon_ticket) {
+                        // We need to set a player starter, otherwise the player won't be able to continue, they can still select a starter though
+                        player.starter(0);
+                      }
+                    }
+                  });
+
+                  // Handle our statistics
+                  Object.entries(data.payload.statistics).forEach(([key, value]) => {
+                    if (typeof value == 'number') {
+                      GameHelper.incrementObservable(App.game.statistics[key], value);
+                      if (!_statistics[key]) _statistics[key] = 0;
+                      _statistics[key] += value;
+                      return;
+                    }
+                    // Array
+                    if (value.constructor.name == 'Array') {
+                      if (!_statistics[key]) _statistics[key] = [];
+                      value.forEach((v, k) => {
+                        if (!v) return;
+                        GameHelper.incrementObservable(App.game.statistics[key][k], v);
+                        if (!_statistics[key][k]) _statistics[key][k] = 0;
+                        _statistics[key][k] += value;
+                      })
+                      return;
+                    }
+                    // Object
+                    if (value.constructor.name == 'Object') {
+                      if (!_statistics[key]) _statistics[key] = {};
+                      if (key == 'routeKills') {
```

Comment:
```suggestion
                      if (keyItem === KeyItems.KeyItem.Dungeon_ticket) {
                        // We need to set a player starter, otherwise the player won't be able to continue, they can still select a starter though
                        player.starter(0);
                      }
                    }
                  });

                  // Handle our statistics
                  Object.entries(data.payload.statistics).forEach(([key, value]) => {
                    if (typeof value === 'number') {
                      GameHelper.incrementObservable(App.game.statistics[key], value);
                      if (!_statistics[key]) _statistics[key] = 0;
                      _statistics[key] += value;
                      return;
                    }
                    // Array
                    if (value.constructor.name === 'Array') {
                      if (!_statistics[key]) _statistics[key] = [];
                      value.forEach((v, k) => {
                        if (!v) return;
                        GameHelper.incrementObservable(App.game.statistics[key][k], v);
                        if (!_statistics[key][k]) _statistics[key][k] = 0;
                        _statistics[key][k] += value;
                      })
                      return;
                    }
                    // Object
                    if (value.constructor.name === 'Object') {
                      if (!_statistics[key]) _statistics[key] = {};
                      if (key === 'routeKills') {
```